lucene/backward-codecs/src/test/org/apache/lucene/index/TestManyPointsInOldIndex.java

```diff
@@ -36,7 +36,7 @@
 //
 // Compile:
 //   1) temporarily remove 'extends LuceneTestCase' above (else java doesn't see our static void main)
```

Comment:
I'd look as to what this class actually does... seems weird. Classpath below (under "run") is wrong for gradle.